docs/TempUG.md

```diff
@@ -545,93 +546,118 @@ Please use show lessonIndex before deleting task!
 
 ### Edit Feature
 
-About the feature (generally that is similar across states)
-
-Format: `command COMPULSORY [optional]` (if same command format across states)
-* Format info 1
-* Format info 2
-
-<box type="tip" seamless> 
-
-**Tips:**
-- Tip 1
-- Tip 2
-
-</box>
-
+Edits the specified item in the student/schedule list of the application.
 
-#### For Student:
-<!-- use -subject SUBJECT for student -->
-Format: `command COMPULSORY [optional]` (for list specific format)
-* Format info 1
-* Format info 2
+#### For student list:
```

Comment:
i think need to change to #### For Student:

Reply:
Required update made. Thank you for pointing that out